addons/test_mail/tests/test_ir_attachment.py

```diff
@@ -0,0 +1,76 @@
+from odoo.addons.mail.tests.common import mail_new_test_user
+from odoo.addons.test_mail.tests.common import TestMailCommon
+from odoo.exceptions import AccessError
+
+class TestAttachment(TestMailCommon):
+
+    def test_attachment_forbid_unlink(self):
+        """Check that removing message attachments is prevented on other user's messages."""
+        test_record = self.env['mail.test.simple'].with_context(self._test_context).create({
+            'name': 'Test',
+            'email_from': 'ignasse@example.com',
+        })
+
+        user_admin, user_employee = self.env['res.users'].search([('login', 'in', ['admin', 'employee'])], order='login asc')
```

Comment:
Can't you simply access "self.user_admin" and "self.user_employee"?

Seems like they are available in setupClass.